oxcaml/tests/backend/regalloc_validator/check_regalloc_validation.ml

```diff
@@ -95,6 +95,7 @@ module Cfg_desc = struct
         ~fun_num_stack_slots:(Stack_class.Tbl.make 0)
         ~fun_poll:Lambda.Default_poll
         ~next_instruction_id:(InstructionId.make_sequence ())
+        ~fun_ret_type:Cmm.typ_val
```

Comment:
I does not matter that much at this
point, but it feels like `fun_ret_type`
should be part of the `t` type above,
so that this line would copy the value
from the passed `t` value.